engine/dlib/src/dlib/lz4.h

```diff
@@ -18,7 +18,7 @@
 #include <stdint.h>
 #include "shared_library.h"
 
-#define DMLZ4_MAX_OUTPUT_SIZE (1 << 30)
+#define DMLZ4_MAX_OUTPUT_SIZE (1LL << 31)  // 2GB limit instead of 1GB
```

Comment:
But maybe we don't need it at all.. 
This workaround added many years ago, not sure if it is still make sense: 
https://github.com/defold/defold/blob/e7d9a387928d23790a199458274fa070a3bcaf88/engine/dlib/src/dlib/lz4.cpp#L24-L37